packages/graphql_server/src/updateProject.ts

```diff
@@ -144,6 +144,54 @@ const updateProjectFounders = async (repoName: string, owner: string) => {
   }
 }
 
+const updateProjectForkHistory = async (repoName: string, owner: string) => {
+  if (!(await repoIsAlreadyInDB(repoName, owner))) {
+    return
+  }
+  //@Todo: get real fork history -> waiting for maxis PR
+  const forkHistory = mockForkHistory
+
+  await updateSupabaseProject(repoName, owner, { fork_history: forkHistory })
+}
+
+/**
+ * Updates the github stats of a repo.
+ * Not the star-history though.
+ * @param {string} repoName - The name of the repo.
+ * @param {string} owner - The name of the owner of the repo.
+ */
+const updateProjectGithubStats = async (name: string, owner: string) => {
+  if (!(await repoIsAlreadyInDB(name, owner))) return
+  const githubStats: GitHubInfo | null = await getGithubData(name, owner)
+  if (!githubStats) {
+    console.log('Could not get github stats for ', name, 'owned by', owner)
```

Comment:
now way to access error from outside of method

Reply:
it is bad design as I realize now but won't change before the midterm review